### PR TITLE
[BACKPORT v1.13] Add PX4 vision v1.5 Airframe

### PR DESCRIPTION
**Pull request by julianoes**
Backport of #20155.

### CODE REVIEW

**Review by Jaeyoung-Lim — APPROVED** [review]

File: boards/cubepilot/cubeorange/test.px4board [hunk 1]
```diff
@@ -12,3 +12,4 @@ CONFIG_MODULES_TEMPERATURE_COMPENSATION=n
 CONFIG_BOARD_TESTING=y
 CONFIG_DRIVERS_TEST_PPM=y
 CONFIG_SYSTEMCMDS_MICROBENCH=y
+CONFIG_SYSTEMCMDS_BL_UPDATE=n
```
Comment (julianoes) [inline boards/cubepilot/cubeorange/test.px4board:15]: @dagar is this ok to save flash for the test platform for the v1.13 backport?

Comment (dagar) [reply boards/cubepilot/cubeorange/test.px4board:15]: Yes that's fine.

**Comment by vincentpoont2** [comment]
https://github.com/PX4/PX4-Autopilot/pull/19990
image
`param set-default SYS_USE_IO 0 ` was removed here. 

I assume it is because `SYS_CTRL_ALLOC` is enable by default in main FW. 
It looks like 1.13.X has `SYS_CTRL_ALLOC `disable by default, so `param set-default SYS_USE_IO 0` is needed due to PX4 vision  motors are connected to AUX (FMU PWM OUT). 

I am guessing we should put `param set-default SYS_USE_IO 0` back to the 1.13 branch? instead of enabling `SYS_CTRL_ALLOC` on this particular airframe.

@julianoes what do you think?

**Comment by julianoes** [comment]
I think you're right @vincentpoont2. Should we make a PR for that so it can go into v1.13.2 at some point?

**Comment by vincentpoont2** [comment]
@julianoes yes, can you please assist on that?

**Comment by julianoes** [comment]
@vincentpoont2 looks like it's there: https://github.com/PX4/PX4-Autopilot/blob/release/1.13/ROMFS/px4fmu_common/init.d/airframes/4016_holybro_px4vision#L21

**Comment by julianoes** [comment]
Oh, you're right, for v1.5 it's missing.

https://github.com/PX4/PX4-Autopilot/pull/20534

**Pull request merged**